### ISSUE TITLE
Remove dark color scheme from tokens package

### ISSUE THREAD

**Issue by sam-b-rose** [Shopify/polaris-react-archive#5994]
Remove dark:

- [x] Remove `colors.dark` from tokens
- [x] Remove scripts and functions for creating multiple schemes

Improve usage:

- [ ] Export tokens key/value pairs as default (no metadata)
- [x] Provide separate export for metadata for improved tree-shaking

**Comment by mathildebuenerd** [comment]
@samrose3 I just came across this change, do you have any context on why this change was introduced and how we now handle dark theme with polaris? Thanks!

**Comment by sam-b-rose** [comment]
Hi @mathildebuenerd! Yes, we removed the dark color scheme support to alleviate technical debt and make an upcoming color palette refactor more feasible.

The dark color scheme wasn't being utilized holistically across the Shopify admin (full-scale dark theme). It was only being used in limited subsections of the UI. We decided to remove the dark color scheme and instead provide a [smaller subset of dark color tokens](https://github.com/Shopify/polaris/blob/main/documentation/guides/migrating-from-v9-to-v10.md#using-dark-color-tokens) to implement the current dark UI.

We have requests to support a dark color scheme and are working towards a better state which will set us up to implement this better than the previous implementation. Hope that context helps!
